src/fcntl.rs

```diff
@@ -182,6 +183,15 @@ pub fn readlinkat<'a, P: ?Sized + NixPath>(dirfd: RawFd, path: &P, buffer: &'a m
     wrap_readlink_result(buffer, res)
 }
 
+/// Computes the raw fd consumed by a function of the form `*at`.
+#[inline]
+pub fn at_rawfd(fd: Option<RawFd>) -> raw::c_int {
```

Comment:
Does this need to be public?  Will Nix consumers ever need to use it directly?

Reply:
If you need to use it from within multiple of Nix's modules, then maybe you should be using `pub(crate)` or `pub(super)`.

Reply:
Oh, I wasn't aware of such a thing. Done.